README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/mendersoftware/mender-convert.svg?branch=master)](https://travis-ci.com/mendersoftware/mender-convert)
+
```

Comment:
I am guessing this badge will work once we merge this PR? As it currently says "unknown"

Reply:
Exactly, it says unknown because the badge points to master where we don't have yet a travis build file. So it should be fine once we merge :)